tests/integration/components/task-card-test.js

```diff
@@ -293,3 +300,48 @@ test('assignment dropdown does not render if user has no ability', function(asse
     assert.equal(user.icon.url, 'test.png');
   });
 });
+
+test('the selected-item component is visable when a task has a user assigned ', function(assert) {
+  assert.expect(5);
+
+  let task = { id: 'task' };
+  let user = { id: 'user', username: 'testuser', photoThumbUrl: 'test.png' };
+  let users = [user];
+  let taskUser = user;
+
+  setProperties(this, { task, users, taskUser });
+
+  renderPage();
+  assert.equal(page.selectedItem.selectedIcon.src, taskUser.photoThumbUrl, 'the selected user photo icon renders');
+  assert.notOk(page.tooltip, 'there is no tooltip because mouse is not hovering.');
```

Comment:
These are valid assertions and I'm completely fine with keeping them in this way. However, a simpler thing you could've also done is simply do

```Javascript
assert.ok(page.selectedItem.isVisible, 'The selected item component is rendered on the card.');
```

The way the component is implemented right now means that both methods work and test the behavior correctly, so as i said, both methods are as valid. 

However, if the `selectedItem` component changes internally at some point the `isVisible` test will still work.